fix_proto_conf_desc.py

```diff
@@ -0,0 +1,100 @@
+import ida_segment
+import idautils
+import idc
+import ida_bytes
+
+def parse_pcd(pcd_addr):
+
+	#create comments for the PCD struct
+	ida_bytes.create_dword(pcd_addr, 4)
+	idc.op_offset(pcd_addr, 0, REF_OFF32|REFINFO_SIGNEDOP, -1, pcd_addr, 0)
+	idc.set_cmt(pcd_addr, "Protocol Descriptor", 0)
+
+	ida_bytes.create_dword(pcd_addr+4, 4)
+	idc.op_offset(pcd_addr+4, 0, REF_OFF32|REFINFO_SIGNEDOP, -1, pcd_addr+4, 0)
+	idc.set_cmt(pcd_addr+4, "Nominal Type Descriptor", 0)
+
+	ida_bytes.create_dword(pcd_addr+8, 4)
+	pwt_val = idc.get_wide_dword(pcd_addr+8)
+	if(pwt_val>0):
+		#create the offset reference only if it is defined
+		idc.op_offset(pcd_addr+8, 0, REF_OFF32|REFINFO_SIGNEDOP, -1, pcd_addr+8, 0)
+	idc.set_cmt(pcd_addr+8, "PWT", 0)
+
+	ida_bytes.create_dword(pcd_addr+12, 4)
+	idc.set_cmt(pcd_addr+12, "Flags", 0)
+
+	flags_value = idc.get_wide_dword(pcd_addr+12)
+	if(flags_value != 0x30000 and flags_value != 0x300c0 and flags_value != 0x30080):
+		print("Unsupported flag value: ",hex(flags_value),". Not parsing this... (WIP!)")
+		return
+
+	ida_bytes.create_dword(pcd_addr+16, 4)
+	base_relwitness_addr = 0
+
+	flag_mask_res = flags_value & 0x000c0
+	if(flag_mask_res == 0xc0):
```

Comment:
remove `()` for `if` statements
```suggestion
	if flag_mask_res == 0xc0:
```